source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/layout/HorizontalFlowLayout.java

```diff
@@ -65,15 +65,28 @@ protected void onMeasure(int widthMeasureSpec, int heightMeasureSpec) {
         // need to call super.onMeasure(...) otherwise we'll get some funny behaviour
         super.onMeasure(widthMeasureSpec, heightMeasureSpec);
 
-        final int width = MeasureSpec.getSize(widthMeasureSpec);
+        int width = MeasureSpec.getSize(widthMeasureSpec);
```

Comment:
@jwoo-msft I believe on ImageSet uses this layout. Are there any other scenarios we should be testing here?

Reply:
Yes, the width and height here are the requested height from the parent. I can update the name.

Reply:
I added a comment to explain the initial values here. We want a generic name since we reassign the variable later.